seatunnel-core/seatunnel-core-starter/src/main/java/org/apache/seatunnel/core/starter/utils/ConfigBuilder.java

```diff
@@ -36,13 +37,16 @@
 import java.util.Map;
 import java.util.Objects;
 import java.util.Optional;
+import java.util.stream.Collectors;
 
 /** Used to build the {@link Config} from config file. */
 @Slf4j
 public class ConfigBuilder {
 
     public static final ConfigRenderOptions CONFIG_RENDER_OPTIONS =
             ConfigRenderOptions.concise().setFormatted(true);
+    public static List<String> KEYWORDS =
```

Comment:
I am not sure whether we need this feature as different key names need to be maintained

Reply:
I think it is necessary to print the configuration log for easy troubleshooting. We can start by maintaining generic keywords such as "password" and "secret".

Reply:
how about reuse https://github.com/apache/seatunnel/blob/764d8b0bc8a12100bf2ef07c9b1488ba2bda70ab/seatunnel-core/seatunnel-core-starter/src/main/java/org/apache/seatunnel/core/starter/utils/ConfigShadeUtils.java#L50 ?

Reply:
configure privacy keywords in seatunnel's profile，i will modified.